pysma/const.py

```diff
@@ -229,65 +235,400 @@
-    SFACTOR: 10,
-}
-SENSOR_BATTERY_TEMP_B = {
-    SKEY: "6100_40495B00_1",
-    SNAME: "battery_temp_b",
-    SUNIT: "C",
-    SFACTOR: 10,
-}
-SENSOR_BATTERY_TEMP_C = {
-    SKEY: "6100_40495B00_2",
-    SNAME: "battery_temp_c",
-    SUNIT: "C",
-    SFACTOR: 10,
-}
-SENSOR_BATTERY_VOLTAGE_A = {
+SENSOR_BATTERY_DC_VOLTAGE_A = { # Example: "6100_00495C00":{"7":[{"val":16363},{"val":null},{"val":null}]} # 10905=Bat.Vol\4807680\Battery voltage
     SKEY: "6100_00495C00_0",
-    SNAME: "battery_voltage_a",
+    SNAME: "battery_dc_voltage_a",
     SUNIT: "V",
     SFACTOR: 100,
 }
-SENSOR_BATTERY_VOLTAGE_B = {
+SENSOR_BATTERY_DC_VOLTAGE_B = { # Example: "6100_00495C00":{"7":[{"val":16363},{"val":null},{"val":null}]} # 10905=Bat.Vol\4807680\Battery voltage
     SKEY: "6100_00495C00_1",
-    SNAME: "battery_voltage_b",
+    SNAME: "battery_dc_voltage_b",
     SUNIT: "V",
     SFACTOR: 100,
 }
-SENSOR_BATTERY_VOLTAGE_C = {
+SENSOR_BATTERY_DC_VOLTAGE_C = { # Example: "6100_00495C00":{"7":[{"val":16363},{"val":null},{"val":null}]} # 10905=Bat.Vol\4807680\Battery voltage
     SKEY: "6100_00495C00_2",
-    SNAME: "battery_voltage_c",
+    SNAME: "battery_dc_voltage_c",
+    SUNIT: "V",
+    SFACTOR: 100,
+}
+SENSOR_BATTERY_DC_CHARGING_VOLTAGE_A = { # Example "6102_00493500":{"7":[{"val":17520},{"val":null},{"val":null}]} # 
+    SKEY: "6102_00493500_0",
+    SNAME: "battery_dc_charging_voltage_a",
+    SUNIT: "V",
+    SFACTOR: 100,
+}
+SENSOR_BATTERY_DC_CHARGING_VOLTAGE_B = { # Example: "6102_00493500":{"7":[{"val":17520},{"val":null},{"val":null}]} # 
+    SKEY: "6102_00493500_1",
+    SNAME: "battery_dc_charging_voltage_b",
+    SUNIT: "V",
+    SFACTOR: 100,
+}
+SENSOR_BATTERY_DC_CHARGING_VOLTAGE_C = { # Example: "6102_00493500":{"7":[{"val":17520},{"val":null},{"val":null}]} # 
+    SKEY: "6102_00493500_2",
+    SNAME: "battery_dc_charging_voltage_c",
     SUNIT: "V",
     SFACTOR: 100,
 }
-SENSOR_BATTERY_CURRENT_A = {
+SENSOR_BATTERY_DC_CURRENT_A = { # Example: "6100_40495D00":{"7":[{"val":44},{"val":null},{"val":null}]} # 
     SKEY: "6100_40495D00_0",
-    SNAME: "battery_current_a",
+    SNAME: "battery_dc_current_a",
     SUNIT: "A",
     SFACTOR: 1000,
 }
-SENSOR_BATTERY_CURRENT_B = {
+SENSOR_BATTERY_DC_CURRENT_B = { # Example: "6100_40495D00":{"7":[{"val":44},{"val":null},{"val":null}]} # 
     SKEY: "6100_40495D00_1",
-    SNAME: "battery_current_b",
+    SNAME: "battery_dc_current_b",
     SUNIT: "A",
     SFACTOR: 1000,
 }
-SENSOR_BATTERY_CURRENT_C = {
+SENSOR_BATTERY_DC_CURRENT_C = { # Example: "6100_40495D00":{"7":[{"val":44},{"val":null},{"val":null}]} # 
     SKEY: "6100_40495D00_2",
-    SNAME: "battery_current_c",
+    SNAME: "battery_dc_current_c",
     SUNIT: "A",
     SFACTOR: 1000,
 }
+SENSOR_BATTERY_AC_POWER_LIMIT = { # Example: "6800_00832A00":{"7":[{"low":3680,"high":3680,"val":3680}]} # 10121=Inverter.WLim\8595968\Maximum active power device
+    SKEY: "6800_00832A00",
+    SNAME: "battery_ac_power_limit",
+    SUNIT: "W",
+}
```

Comment:
The sensor is called Inverter_WLim by SMA. Maybe `SENSOR_INVERTER_POWER_LIMIT` would be a better name, as it is related to the inverter and not the battery?